src/lib/data.ts

```diff
@@ -218,26 +247,26 @@ async function get_test_metadata(dir_name: string): Promise<TestData[]> {
             normalizeTags : true,
             explicitArray : true,
         });
-        const metadata = package_js.package.metadata[0]
+        const test_metadata = package_js.package.metadata[0]
     
-        const alternate_title = metadata.meta.find((entry: any): boolean => entry["$"].property === "dcterms:alternative");
-        const final_title = alternate_title === undefined ? get_string_value("dc:title", "(No title)") : alternate_title._;
+        // const alternate_title = get_single_meta_value("dcterms:alternative");
+        // const final_title = alternate_title === undefined ? get_string_value("dc:title", "(No title)") : alternate_title._;
```

Comment:
These are commented out; looks like you moved them to methods. Delete, I guess? Fix in some way.

Reply:
Oops. I meant to remove them... thanks.